newIDE/app/src/ObjectsRendering/ObjectsRenderingService.js

```diff
@@ -15,6 +15,7 @@ import * as PIXI from 'pixi.js-legacy';
 import optionalRequire from '../Utils/OptionalRequire';
 const path = optionalRequire('path');
 const electron = optionalRequire('electron');
+global.PIXI = PIXI;
```

Comment:
Can you remove this?

Reply:
you told me to add this. If I remove it I get
`index.js:1 Unable to load module "pixi-tilemap/dist/pixi-tilemap.umd". Only files located in D:\DEV\GD\GDevelop\newIDE\app\resources\GDJS\Runtime\Extensions\TileMap are supported. Exception was: ReferenceError: PIXI is not defined`

and the ide crashes

Reply:
Any ideas if there are any other ways around this? pixi-tilemap will crash Gdevelop without PIXI being global

Reply:
This is because `pixi-tilemap.umd` is still not a proper UMD function as it references `PIXI` without taking it as a dependency. Let's keep this then, but add a comment: ` // Expose PIXI as a global variable as pixi-tilemap.umd reference it without taking it as a dependency.`

Reply:
yeah, my other plugins still have same requirement, just in case you use pixi-spine

Reply:
Thanks for confirming Ivan! We'll keep this then :) 
We're not using pixi-spine yet but I think this is something that could be of interest later 👀